.github/workflows/build-push-erigon.yml

```diff
@@ -29,6 +29,14 @@ on:
 jobs:
   erigon:
     runs-on: ${{ inputs.runs_on }}
+    strategy:
+      matrix:
+        platform: [linux/amd64, linux/arm64]
+      fail-fast: false
+    strategy:
+      matrix:
+        platform: [linux/amd64, linux/arm64]
```

Comment:
we're not building for linux ARM ser :D
i think its `darwin/arm64`?

Reply:
i prefer linuxism over darwinism m8